### PR TITLE
RATEST: Rename packages and update the read me file

### PR DESCRIPTION
**Pull request by kdaud**
Goal:
Rename packages and update the read me file

### CODE REVIEW

**Comment by kdaud** [comment]
@sherrif10  take a look at this PR. I see the build failing as a result of the the Login test as per the cucumber report and we need to fix this probably.

**Comment by sherrif10** [comment]
@kdaud  sure will share a fix for this soon

**Comment by kdaud** [comment]
@sherrif10 what do you say about this PR?

**Comment by kdaud** [comment]
@sherrif10 what do you say about this PR?

**Comment by kdaud** [comment]
@sherrif10 what do you say about this PR?

**Comment by sherrif10** [comment]
Do you mind fixing the login failure as well.

**Comment by sherrif10** [comment]
i also left a comment whether there is a difference btn palladium and kenyaEmr.

**Comment by kdaud** [comment]
> Do you mind fixing the login failure as well.

I will fix the the build failure over the weekend in another PR!

**Comment by sherrif10** [comment]
Sure will also try find time and fix it.

**Comment by kdaud** [comment]
@sherrif10 is this commit worth master/

File: src/test/java/org/openmrs/kenyaemr/contrib/qaframework/RunTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.openmrs.contrib.kenyamer.qaframework;
+package org.openmrs.kenyaemr.contrib.qaframework;
```
Comment (sherrif10) [inline src/test/java/org/openmrs/kenyaemr/contrib/qaframework/RunTest.java:1]: Thanks @kdaud

File: src/test/java/org/openmrs/kenyaemr/contrib/qaframework/RunTest.java [hunk 2]
```diff
@@ -8,9 +8,9 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
         // defining ordered by name features folder
-        features = { "src/features" },
+        features = { "src/test/resources/features" },
         // defining the definition steps package
-        glue = "org.openmrs.contrib.kenyaemr.qaframework.automation", plugin = { "html:target/index.html",
+        glue = "org.openmrs.palladiumkenya.contrib.qaframework.automation", plugin = { "html:target/index.html",
```
Comment (sherrif10) [inline src/test/java/org/openmrs/kenyaemr/contrib/qaframework/RunTest.java:13]: Did you forgot to change this.

Comment (kdaud) [reply src/test/java/org/openmrs/kenyaemr/contrib/qaframework/RunTest.java]: Thanks for the catch @sherrif10, it was an oversight!

**Comment by kdaud** [comment]
@sherrif10 is there recommendation concerning this PR?

**Comment by kdaud** [comment]
closing this PR and opened a new one in `pull/10`

**Pull request closed without merging**